src/sidebar/services/annotations-exporter.tsx

```diff
@@ -152,6 +153,90 @@ export class AnnotationsExporter {
     return `${headers}\n${annotationsContent}`;
   }
 
+  buildHTMLExportContent(
+    annotations: APIAnnotationData[],
+    {
+      groupName = '',
+      displayNamesEnabled = false,
+      defaultAuthority = '',
+      /* istanbul ignore next - test seam */
+      now = new Date(),
+    }: ExportOptions = {},
+  ): string {
+    const { uri, title, uniqueUsers, replies, extractUsername } =
+      this._exportCommon(annotations, {
+        displayNamesEnabled,
+        defaultAuthority,
+      });
+
+    return renderToString(
+      <html lang="en">
+        <head>
+          <title>Annotations export - Hypothesis</title>
+          <meta charSet="UTF-8" />
+        </head>
+        <body>
+          <header>
+            <p>
+              <time dateTime={now.toISOString()}>{formatDateTime(now)}</time>
+            </p>
+            <p>
+              <strong>{title}</strong>
+            </p>
+            <p>
+              <a href={uri} target="_blank" rel="noopener noreferrer">
+                {uri}
+              </a>
+            </p>
+
+            <dl>
+              <dt>Group</dt>
+              <dd>{groupName}</dd>
+              <dt>Total users</dt>
+              <dd>{uniqueUsers.length}</dd>
+              <dt>Users</dt>
+              <dd>{uniqueUsers.join(', ')}</dd>
+              <dt>Total annotations</dt>
+              <dd>{annotations.length}</dd>
+              <dt>Total replies</dt>
+              <dd>{replies.length}</dd>
+            </dl>
+          </header>
+          <hr />
+          <section>
+            {annotations.map((annotation, index) => {
+              const page = pageLabel(annotation);
+              return (
+                <Fragment key={annotation.id}>
+                  <article>
+                    <strong>Annotation {index + 1}:</strong>
```

Comment:
Per notes on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article, things that look like headings should really use an `<hN>` tag.